docs/resources/evs_volume.md

```diff
@@ -128,3 +133,21 @@ Volumes can be imported using the `id`, e.g.
 ```
 $ terraform import huaweicloud_evs_volume.volume_1 14a80bc7-c12c-4fe0-a38a-cb77eeac9bd6
 ```
+Note that the imported state may not be identical to your resource definition, due to some attrubutes missing from the API response, security or some other reason. The missing attributes include: cascade. It is generally recommended running terraform plan after importing an evs volume. You can then decide if changes should be applied to the evs volume, or the resource definition should be updated to align with the evs volume. Also you can ignore changes as below.
```

Comment:
Note that the imported state may not be identical to your resource definition, since the `cascade` is only used for destroying.
It is generally ...